client/src/components/events-components/EventCreation.tsx

```diff
@@ -0,0 +1,110 @@
+import type { EventType } from "@/utils/interfaces";
+import { Card, CardContent, CardTitle } from "../ui/card";
+import { Calendar } from "../ui/calendar";
+import { useEffect, useState } from "react";
+import { Button } from "../ui/button";
+import { Input } from "../ui/input";
+import {
+  Select,
+  SelectContent,
+  SelectTrigger,
+  SelectValue,
+  SelectItem,
+} from "../ui/select";
+
+interface EventCreationProp {
+  event?: EventType;
+}
+
+const sportsOptions = [
+  "SOCCER",
+  "FOOTBALL",
+  "BASKETBALL",
+  "BASEBALL",
+  "TENNIS",
+  "PICKLEBALL",
+  "SOFTBALL",
+  "RACQUETBALL",
+  "FRISBEE",
+  "VOLLEYBALL",
+  "GOLF",
+  "HOCKEY",
+];
+
+const EventCreation = ({ event }: EventCreationProp) => {
+  const [description, setDescription] = useState("");
+  const [sport, setSport] = useState("");
+  const [eventImage, setEventImage] = useState("");
+  const [eventTime, setEventTime] = useState("12:00");
+  const [eventLocation, setEventLocation] = useState("");
+  const [date, setDate] = useState<Date | undefined>(new Date());
+
+  useEffect;
+  const handleEventCreation = () => {
+    console.log(event);
```

Comment:
understand it's just a placeholder, but let's avoid introducing `console.log` in PR, probably commit only when the functionality is finished.